src/main.cpp

```diff
@@ -168,7 +168,7 @@ namespace
 			.key_down =
 				[&](int key) noexcept
 			{
-				log("key down: "sv, key);
+				// log("key down: "sv, key);
```

Comment:
This probably wasn't meant to be commented out.

```suggestion
				log("key down: "sv, key);
```